### DISCUSSION TITLE
create formula to build and copy to Application

### DISCUSSION THREAD

**Discussion by yulrizka** [Homebrew/discussions#63] [Everyday usage] [answered]
I created an opensource osx application but I did not join apple developer program. 
And some time it's hard to run it in Catalina for example. At least in my current laptop
there is no option to run unsigned app. 

To make it easier i want to create a formula to build the app using `xcodebuild`. 
I managed to do that, but I wonder how to copy the `XXX.app` to '/Application`?
Is there maybe a better way to do it?

**Comment by SMillerDev** [comment, marked as answer]
I'd recommend adding a caveat to tell users to make a symlink. Formulae really only ever install to the designated location.
